### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

Add instruction to install from conda and tweaked instruction a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,16 @@ python -m http.server 8001
...
-Run
+To install:
 ```
 pip install git+https://github.com/jupyterhub/nbserverproxy
+```
+or 
+```
+conda install -c conda-forge nbserverproxy
+```
+Enable extension by running:
```
Comment (betatim) [inline README.md]: I've seen other extensions that did not need enabling anymore when they were installed via conda. Do you know what needs to happen on the conda package side for that to be the case here as well?

**Comment by betatim** [comment]
Thanks for contributing and looks good to me. Let's figure out the enabling of extensions bit and then merge.

File: README.md (hunk 1 above)
Comment (lsetiawan) [reply README.md]: I'm working on enabling this on the conda side. Stay tuned. Thanks.

**Comment by lsetiawan** [comment]
Sorry all for the late update: But I've tested the extensions via conda install, and all seems to work. Thanks.

https://anaconda.org/conda-forge/nbserverproxy

**Comment by betatim** [comment]
What was your thinking behind removing the `pip` based install instructions? I think keeping them would be useful for those who don't have conda to remind them that that they have to do more than just `pip install`.

**Comment by lsetiawan** [comment]
@betatim Apologies. 

> I think keeping them would be useful for those who don't have conda to remind them that that they have to do more than just `pip install`.

Yea that makes sense. I've added it back. Thanks. Hope that works.

**Comment by betatim** [comment]
No worries :) This is why we talk about things.

Merging now and congratulations on your first contribution to this project!

**Pull request merged**